core-web/libs/portlets/dot-analytics/data-access/jest.config.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
+export default {
+    displayName: 'analytics-data-access',
```

Comment:
@oidacra I suggest using the prefix:  "portlet" like `portlets-dot-analytics` In this case `portlets-dot-analytics-data-access` yeah, so long, but to follow the pattern